prime-router/src/main/kotlin/Report.kt

```diff
@@ -879,5 +876,9 @@ class Report : Logging {
                 default
             }
         }
+
+        private fun Row.getStringOrNull(columnName: String): String? {
```

Comment:
I did not see this function being used.  Remove?

Reply:
It's used in the Report class itself to safely query each row for the schema values if they exist.